src/main/java/com/twilio/sdk/resource/instance/Account.java

```diff
@@ -846,6 +847,181 @@ public UsageRecordList getUsageRecords() {
 	}
 
 	/**
+	 * Returns multiple Usage Records for each usage category,
+	 * each representing usage over a daily time-interval
+	 *
+	 * @return list of Usage Records over a daily time-interval
+	 */
+	public UsageRecordList getDailyUsageRecords() {
```

Comment:
for curiosity sake, why do we need one per time frame as opposed to a generic method where the user can specify the time frame? 

even if we leave these, can we do `getUsageRecordsForPeriod(Map<String, String> filters, Type type)` and have all these call into that one?